test/modules/storage-aws-s3/backup_storage_test.go

```diff
@@ -0,0 +1,228 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2022 SeMI Technologies B.V. All rights reserved.
+//
+//  CONTACT: hello@semi.technology
+//
+
+package test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/semi-technologies/weaviate/entities/backup"
+	"github.com/semi-technologies/weaviate/modules/storage-aws-s3/s3"
+	"github.com/semi-technologies/weaviate/test/docker"
+	moduleshelper "github.com/semi-technologies/weaviate/test/helper/modules"
+	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_S3Storage_Backup(t *testing.T) {
+	ctx := context.Background()
+	compose, err := docker.New().WithMinIO().Start(ctx)
+	if err != nil {
+		panic(errors.Wrapf(err, "cannot start"))
```

Comment:
Why don't you use t.Fatal() instead of panic()?
 it is made this

Reply:
fixed